AnnoDesigner/Models/UserDefinedColorType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnnoDesigner.Models
+{
+    public enum UserDefinedColorType
```

Comment:
Might be worth adding a comment here that a string value of `UserDefinedColorType` is used for localization.